pkg/crypto/tinkcrypto/primitive/composite/ecdhes/ecdhes_key_template.go

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
what’s a comparable template in Tink that does key exchange / wrapping?

Reply:
hybrid's [ECIESHKDFAES128CTRHMACSHA256KeyTemplate](https://github.com/google/tink/blob/master/go/hybrid/hybrid_key_templates.go#L45)